packages/open-next/src/core/routing/matcher.ts

```diff
@@ -229,7 +229,7 @@ export function handleRewrites<T extends RewriteDefinition>(
     }
     rewrittenUrl = isExternalRewrite
       ? `${protocol}//${rewrittenHost}${rewrittenPath}`
-      : `/${rewrittenPath}`;
+      : `${rewrittenPath}`;
```

Comment:
Are we sure we can trust `rewrittenPath` will never have a leading slash? If no, maybe we need a util function that strips the final `url` from redundant slashes.

I just wanted to make sure we don't play whack-a-mole with these kind of things.

Reply:
It does always have one now actually. This line has changed https://github.com/opennextjs/opennextjs-aws/blob/2c2b3438a21f15981deeb3e3cfc82a0691879c1c/packages/open-next/src/core/routing/util.ts#L50

The only things where i could see an error here is if the url itself is broken, so it would be a user error.

I made the change so that the pathname from `getUrlPart` always start with a leading slash (or is empty).
Before this change pathname would have a leading slash when external and none when internal